test/functionalTest/cases/1169_uri_param_options_map/uri_param_options_map.test

```diff
@@ -0,0 +1,155 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+URI Param OPTIONS
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. OK: Send a request with one valid option (count)
+# 02. OK: Send a request with two valid options (count + normalized)
+# 03. Error: Send a request with one invalid option (canonical)
+# 04. OK: Send a request with ALL valid options (count, normalized, values & text)
+# 05. Error: Send a request with ALL valid options and ONE invalid
+# 06. Error: Test an arbitrary API v2 path
+#
+
+echo "01. OK: Send a request with one valid option (count)"
+echo "===================================================="
+orionCurl --url /v2/entities?options=count --json --payload '{ "id": "ID", "type": "type" }'
+echo
+echo
+
+
+echo "02. OK: Send a request with two valid options (count + normalized)"
+echo "=================================================================="
+orionCurl --url /v2/entities?options=count,normalized --json --payload '{ "id": "ID2", "type": "type" }'
```

Comment:
This will result in an error if at the end normalized mode is implemented for POST /v2/entities, as `{ "id": "ID2", "type": "type" }` is not normalized. However, I understand that will be easy to find&fix at that moment.

NTC